study-builder/studies/brugada/activities/quality-of-life.conf

```diff
@@ -0,0 +1,1296 @@
+  "activityCode": "QUALITY_OF_LIFE",
+  "versionTag": "v1",
+  "displayOrder": 5,
+  "writeOnce": true,
+  "hideExistingInstancesOnCreation": true,
+  "translatedNames": [
+    {
+      "language": "en",
+      "text": ${i18n.en.QOL.name}
+    }
+  ],
+  "translatedSecondNames": [
+    {
+      "language": "en",
+      "text": ${i18n.en.QOL.name}
+    }
+  ],
+  "translatedTitles": [
+    {
+      "language": "en",
+      "text": ${i18n.en.QOL.title}
+    }
+  ],
+  "translatedSummaries": [
+    {
+      "statusCode": "CREATED",
+      "language": "en",
+      "text": ${i18n.en.QOL.summary_created}
+    },
+    {
+      "statusCode": "IN_PROGRESS",
+      "language": "en",
+      "text": ${i18n.en.QOL.summary_in_progress}
+    },
+    {
+      "statusCode": "COMPLETE",
+      "language": "en",
+      "text": ${i18n.en.QOL.summary_complete}
+    }
+  ],
+  "sections": [
+    {
+      "nameTemplate": {
+        "templateType": "TEXT",
+        "templateText": "1",
+        "variables": []
+      },
+      "icons": [],
+      "blocks": [
+        {
+          "titleTemplate": null,
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """
+              <p>$description</p>
+            """,
+            "variables": [
+              {
+                "name": "description",
+                "translations": [
+                  {
+                    "language": "en", "text": ${i18n.en.QOL.survey_description}
+                  }
+                ]
+              }
+            ]
+          },
+          "blockType": "CONTENT",
+          "shownExpr": null
+        },
+        {
+          "blockType": "QUESTION",
+          "shownExpr": null,
+          "question": {
+            include required("../../snippets/picklist-question-single-list.conf"),
+            "stableId": "QOL_HAVE_GENE",
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": """
+                <h3 class="activity-question-title">$q1_prompt</h3>
+              """,
+              "variables": [
+                {
+                  "name": "q1_prompt",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": ${i18n.en.QOL.q1_prompt},
+                    },
+                  ],
+                },
+              ],
+            },
+            include required("../snippets/picklist-1-to-6.conf")
+          },
+        },
+        {
+          "blockType": "QUESTION",
+          "shownExpr": null,
+          "question": {
+            include required("../../snippets/picklist-question-single-list.conf"),
+            "stableId": "QOL_GENE_IS_ALWAYS_ON_MY_MIND",
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": """
+                <h3 class="activity-question-title">$q2_prompt</h3>
```

Comment:
Downside of using names like Q2, Q3, ... if we need to insert a new question before or inbetween, this is not Q2 or Q3 any more. 
Would be nice to use proper string to reflect what the question or prompt is about like we have in existing studies

Reply:
I have removed this file since it is not used anymore